### PR TITLE
Added the new issuer logo ( FlyDesier ).

### PR DESCRIPTION
**Pull request by dc-andysign**
### What

For story - [F2P-66](https://digicatapult.atlassian.net/browse/F2P-66)

Remove company old logos and add new logo.

### Validation

- [x] - issuer logo
- [x] - replace text in readme and other places
- [x] - version incrementation

### CODE REVIEW

File: src/components/Logo/Logo.js [hunk 1]
```diff
@@ -8,14 +8,120 @@ export default function Logo() {
       {/* <i className="fa d-inline fa-lg fa-stop-circle-o" /> */}
       <div className="float-left ml-1 mr-0" style={{ width: 35 }}>
         <svg
```
Comment (n3op2) [inline src/components/Logo/Logo.js:10]: why not saving file as svg?

Comment (dc-andysign) [reply src/components/Logo/Logo.js:10]: From what I remember whenever you take assets and you insert them into your page it's always a bit tricky if what you are trying to add is not a .js file. Therefore it's all in there... There might be better ways, I'm sure ...

**Review by n3op2 — DISMISSED** [review]

**Comment by jonmattgray** [comment]
Version needs bumping to `0.2.4`

**Comment by dc-andysign** [comment]
> Version needs bumping to `0.2.4`

Indeed... changing it... DONE, all fixed!

**Review by mattdean-digicatapult — APPROVED** [review]

**Review by jonmattgray — APPROVED** [review]

**Pull request merged**